packages/file-search/src/browser/quick-editors-open.ts

```diff
@@ -0,0 +1,176 @@
+import { Command } from '@theia/core/lib/common';
+import * as fuzzy from 'fuzzy';
+import { CancellationTokenSource } from '@theia/core/lib/common';
+import { EditorWidget } from '@theia/editor/lib/browser';
+
+export const SHOW_ALL_OPENED_EDITORS: Command = {
+    id: 'workbench.action.showAllEditors',
+    category: 'View',
+    label: 'Show All Opened Editors'
+};
+
+@injectable()
+export class QuickEditorsOpenService implements QuickOpenModel, QuickOpenHandler {
+
+    @inject(ApplicationShell)
+    protected readonly applicationShell: ApplicationShell;
+
+    @inject(FileSystem)
+    protected readonly fileSystem: FileSystem;
+
+    @inject(WorkspaceService)
+    protected readonly workspaceService: WorkspaceService;
+
+    @inject(OpenerService)
+    protected readonly openerService: OpenerService;
+
+    @inject(PrefixQuickOpenService)
+    protected readonly prefixQuickOpenService: PrefixQuickOpenService;
+
+    @inject(MessageService)
+    protected readonly messageService: MessageService;
+
+    @inject(LabelProvider)
+    protected readonly labelProvider: LabelProvider;
+
+    readonly prefix: string = 'edt ';
+
+    get description(): string {
+        return 'Show All Opened Editors';
+    }
+
+    getModel(): QuickOpenModel {
+        return this;
+    }
+
+    getOptions(): QuickOpenOptions {
+        return {
+            placeholder: '',
+            fuzzyMatchLabel: {
+                enableSeparateSubstringMatching: true
+            },
+            fuzzyMatchDescription: {
+                enableSeparateSubstringMatching: true
+            },
+            showItemsWithoutHighlight: true,
+            onClose: () => {
+                this.cancelIndicator.cancel();
+            }
+        };
+    }
+
+    isEnabled(): boolean {
+        return this.workspaceService.opened;
+    }
+
+    open(): void {
+        this.prefixQuickOpenService.open(this.prefix);
+    }
+
+    private cancelIndicator = new CancellationTokenSource();
+
+    public async onType(lookFor: string, acceptor: (items: QuickOpenItem[]) => void): Promise<void> {
+        this.cancelIndicator.cancel();
+        this.cancelIndicator = new CancellationTokenSource();
+        const token = this.cancelIndicator.token;
+
+        const editorItems: QuickOpenItem[] = [];
+
+        // Get URIs of all EditorWidgets that are currently opened
+        // Sorted by URI
+        const widgets = this.applicationShell.widgets
+            .filter((widget): widget is EditorWidget => (widget instanceof EditorWidget))
+            .map(w => w.getResourceUri()!)
+            .sort();
+
+        if (widgets.length === 0) {
+            editorItems.push(new QuickOpenItem({
+                label: 'List of opened editors is currently empty',
+                run: () => false
+            }));
+            acceptor(editorItems);
+            return;
+        }
+
+        for (const uri of widgets) {
+            if (uri.scheme === 'file' && fuzzy.test(lookFor, uri.toString())) {
```

Comment:
We shouldn't need the `fuzzy` test at this point.
The reason the items were still displaying is due to the option [`showItemWithoutHighlight`](https://github.com/theia-ide/theia/pull/5478/files#r293990391)